packages/worker-api/src/interface.ts

```diff
@@ -32,6 +33,15 @@ export interface TrustedCallResult {
   status?: TrustedOperationStatus,
 }
 
+// All the assets we support and string variations of it.
+export type AssetIdStr = "USDT" | "usdt" | "USDC" | "usdc" | "USDC.e" | "usdc.e" | "ETH" | "eth" | "WETH" | "weth"
+
+// If it is a string, we assume that it is base58 encoded.
+export type ShardIdentifierArg = string | ShardIdentifier | EnclaveFingerprint | H256 | Hash;
+
+// If it is a string, we assume that it is base58 encoded.
+export type MrenclaveArg = string | ShardIdentifier | EnclaveFingerprint | H256 | Hash;
+
 export interface ISubmittableGetter<W extends IWorkerBase, Type> {
```

Comment:
This is the supported inputs for the assets.

Also, as described in the PR, the shard/mrenclave argument can now be either a string, or one of these types.